permissions/plugin-configuration-as-code.yml

```diff
@@ -11,7 +11,9 @@ developers:
 - "timja"
 security:
   contacts:
-    jira: "jetersen"
+    jira: 
+       - "jetersen"
+       - "foundation_security_members"
```

Comment:
your format wasn't valid
```suggestion
    jira: "foundation_security_members"
```

Reply:
Thanks for the heads up

Reply:
Is it possible to have both?

Reply:
no you can't have both without re-working the code, it only handles one.